### PR TITLE
headings.md - improve case section and add verb tense section

### PR DESCRIPTION
**Pull request by steroman**
This PR updates the sentence case section and adds a new section about the use of imperative tense in headings.md

### CODE REVIEW

File: docs/headings.md [hunk 1]
```diff
@@ -29,19 +23,35 @@ Proper names or code snippets:
...
 # `String`
 
 # Init
 ```
 
+## Verb tense
+
+Use the imperative form, especially when writing the title of a task. It increases findability.
+In other words, avoid the gerundive form. Users search for "create something", not "creating something".
+
```
Comment (svx) [inline docs/headings.md:42]: Can you change that to one sentence per line?

**Comment by svx** [comment]
Thanks, added one comment, also just to let you know :) https://github.com/ocular-d/styleguide-editorial/tree/main will be the new "master" branch.

ETA is sometime this week for the switch :)

File: docs/headings.md (hunk 1 above)
Comment (steroman) [reply docs/headings.md:42]: done :)

**Review by svx — APPROVED** [review]
Awesome! - Thanks!!!

**Comment by svx** [comment]
@steroman Awesome! I will cherry-pick that PR into the new main branch ASAP!

**Pull request merged**